.github/workflows/ci.yaml

```diff
@@ -67,3 +67,18 @@ jobs:
           printf '%s\n' "$TEMPORAL_TLS_CERT_CONTENT" >> client.crt
           printf '%s\n' "$TEMPORAL_TLS_KEY_CONTENT" >> client.key
           go run ./cmd/temporal workflow list --limit 2
+
+  test-status:
```

Comment:
At least the server repo, we have a step to aggregate the tests results.

Reply:
I don't think we need this—we've probably spent more time talking about whether we need it or not than what it would actually take over the next few years to tweak the branch-protection rules each time the list of OSes changes.

Also, I kinda like being able to see the broken-out status of each platform on the PR itself; there's less digging when something breaks.

Reply:
Removed the `test-status` job.